projects/ng-imgix/src/lib/ix-srcset.directive.spec.ts

```diff
@@ -11,11 +11,13 @@ export class MockElementRef extends ElementRef {
 
 @Component({
   template: `
-  <img src="https://assets.imgix.net/examples/bluehat.jpg" />
-  <img src="https://assets.imgix.net/unsplash/motorbike.jpg" [ixSrcset]="{ minWidth: 800, maxWidth: 1200 }" />
-  <img src="https://assets.imgix.net/unsplash/bridge.jpg?w=500" ixSrcset />
-  <img src="https://assets.imgix.net/unsplash/raspberries.jpg?w=1000" [ixSrcset]="{ disableVariableQuality: true }" />
-  `
+  <div style="position: relative; overflow: hidden; display: inline-block; width: 10px; height: 10px;">
```

Comment:
Nit: is this meant to have a width and a height of 10 for 4 child images? 👀

Reply:
This was more for cleanliness since some pretty large images are shown on screen when Karma runs. I believe the gatsby tests do the same thing as well.